src/main/java/org/jitsi/jigasi/transcription/LocalTxtTranscriptHandler.java

```diff
@@ -190,7 +205,11 @@ public void publish(ChatRoom chatRoom, TranscriptionResult result)
         String name = result.getName();
         String transcription = result.getAlternatives().iterator()
             .next().getTranscription();
-
+        if(!getTargetLanguage().isEmpty())
+        {
+            transcription
+                    = translationService.translate(transcription, "en","hi");
```

Comment:
Any reason to hard-code "hi" here instead of using the value of `getTargetLanguage`?